### DISCUSSION TITLE
Email setup

### DISCUSSION THREAD

**Discussion by arelenglish** [pay-rails/pay#571] [Q&A] [answered]
I noticed that email sent with the Pay Gem don't have a from address set by default, which makes sense. How would the gem know what email to send from. I'd like to submit a PR to clear up the documentation, but wanted clarify first that it is required to create `config/initializers/pay.rb` then add at least the following:

```ruby
Pay.setup do |config|
  config.support_email = "support@coolapplication.com"
end 
```

Are other variables required to be set? I see that business name and address are in the mailer, but assume they would just not render anything if blank.

**Comment by excid3** [comment, marked as answer]
https://github.com/pay-rails/pay/blob/master/app/mailers/pay/application_mailer.rb#L3

**Reply by arelenglish** [reply]
Right, so if that's not set in the initializer, your emails won't be able to send because from will be null. The docs read as if adding the initializer is optional.

**Reply by excid3** [reply]
We can update it to fallback to the ActionMailer default from address. Want to make a PR for that?

**Reply by arelenglish** [reply]
Sure! I'll do it soon!

**Reply by excid3** [reply]
Awesome, thanks! 😄

**Reply by arelenglish** [reply]
And for anyone who comes here, PR is here: https://github.com/pay-rails/pay/pull/587
